java/ql/test/library-tests/dataflow/capture/B.java

```diff
@@ -248,4 +248,15 @@ void run() {
     sink(l.get(0)); // $ hasValueFlow=src
     sink(l2.get(0)); // $ hasValueFlow=src
   }
+
+  void testInstanceInitializer() {
+    String s = source("init");
```

Comment:
Add a comment noting that this is exercising specifically the obinit routine propagating taint?